recipes/cytosim/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cytosim" %}
+{% set version = "0.0.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8234cbbd89fb897915abdabbefb80490e8044c92118363c002a1214ebd6fe01b
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - cmake >=3.14
+    - make 
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - libblas
+    - liblapack
+    - pybind11
+  host:
+    - setuptools
+    - python
+    - libblas
+    - liblapack
+    - pybind11
+  run:
+    - python
+    - libblas
+    - liblapack
+    - pybind11
+
+test:
+  imports:
+    - cytosim
+about:
+  home: The package home page
```

Please add a proper URL

```suggestion
  home: The package home page
```

---

Solved with package homepage